app/components/works/work_type_modal_component.html.erb

```diff
@@ -65,8 +65,8 @@
 
           <div class="mb-4 subtype-container" data-work-type-target="moreTypes" hidden></div>
 
-          <button type="submit" 
-            class="btn btn-primary mx-auto d-block" 
+          <button type="submit"
+            class="btn btn-primary mx-auto d-block"
```

Comment:
Is this a whitespace-only change?

Reply:
looks like it, possibly came from the rebase I just did?

looks like the a cypress test is failing, and may need some attention too (assuming this PR is still relevant after 9 months)